apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -6698,6 +6698,22 @@
         "mainCta": "Sync your accounts",
         "secondCta": "Already created a key?"
       }
+    },
+    "deviceSelection": {
+      "title": "Choose the Ledger device you will use to secure your backup",
+      "description": "You can retrieve your backup by simply connecting your Ledger to the Ledger Live mobile and desktop apps."
+    },
+    "deviceAction": {
+      "title": "Continue on your Ledger {{wording}}",
+      "description": "Follow the instruction which appear on your Ledger’s Trusted Display."
+    },
+    "success": {
+      "sync": "Synchronization successful!",
+      "syncDesc": "Changes in your accounts will now automatically appear across all apps and platforms.",
+      "activation": "Success!",
```

Comment:
i find the `!`s a bit odd. is this really in the designs?

Reply:
https://www.figma.com/design/uI7PKnUK8nAZ0USyiEF3wQ/Wallet-Sync?node-id=6893-18566&t=N4KOimwx2hBW4G5p-0

yup